resources/langs/nheko_en.ts

```diff
@@ -634,11 +634,16 @@
     </message>
     <message>
         <location line="+2"/>
-        <source>Unbanned %1</source>
-        <translation>Unbanned %1.</translation>
+        <source>Unbanned %1.</source>
+        <translation type="unfinished">Unbanned %1..</translation>
```

Comment:
That seems like a period too many. Does the translation update try to fix translations manually and add them everywhere?